crates/rpc-replay/src/block_context.rs

```diff
@@ -1,15 +1,19 @@
-use blockifier::block::{BlockInfo, GasPrices};
+use blockifier::blockifier::block::{BlockInfo, GasPrices};
+use blockifier::bouncer::BouncerConfig;
```

Comment:
What is a Bouncer?

Reply:
If I understand it right, it's a mechanism used to determine how many txs can go in a block depending on different factors like gas, builtin usage, number of events etc.